### PR TITLE
Release 3.0.0-rc.5

### PR DESCRIPTION
**Pull request by bassosimone**

### CODE REVIEW

File: nettests/psiphon.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	)
+	if err != nil {
+		return err
+	}
+	return ctl.Run(builder, []string{""})
+}
+
+// PsiphonTestKeys contains the test keys
+type PsiphonTestKeys struct {
+	IsAnomaly bool `json:"-"`
```
Comment (hellais) [inline nettests/psiphon.go]: I think we should also expose as a test key:
```
        "bootstrap_time": 6.076582013,
        "failure": null,
```

File: internal/log/handlers/cli/results.go [hunk 2]
```diff
@@ -69,6 +69,13 @@ var summarizers = map[string]func(uint64, uint64, string) []string{
 			"",
 		}
 	},
+	"circumvention": func(totalCount uint64, anomalyCount uint64, ss string) []string {
+		return []string{
+			fmt.Sprintf("Detected: %v", anomalyCount > 0),
```
Comment (hellais) [inline internal/log/handlers/cli/results.go]: This should probably say something like: "%d tested" and "%d blocked".

File: nettests/psiphon.go (hunk 1 above)
Comment (bassosimone) [reply nettests/psiphon.go]: https://github.com/ooni/probe-cli/issues/96

File: internal/log/handlers/cli/results.go (hunk 2 above)
Comment (bassosimone) [reply internal/log/handlers/cli/results.go]: https://github.com/ooni/probe-cli/issues/97

File: nettests/psiphon.go (hunk 1 above)
Comment (bassosimone) [reply nettests/psiphon.go]: See #96

**Pull request merged**